wls-wahlvorstand-service/src/test/resources/http.request/wahlvorstand.http

```diff
@@ -0,0 +1,69 @@
+### get token for wls_all_bwb
+POST {{ SSO_URL }}/auth/realms/wls_realm/protocol/openid-connect/token
+Content-Type: application/x-www-form-urlencoded
+
+password = test &
+grant_type = password &
+client_secret = top-secret &
+client_id = wls &
+username = wls_all_bwb
+
+> {%
+    client.global.set("auth_token", response.body.access_token);
+    client.global.set("token_type", response.body.token_type);
+%}
+
+### get token for wls_all_uwb
+POST {{ SSO_URL }}/auth/realms/wls_realm/protocol/openid-connect/token
+Content-Type: application/x-www-form-urlencoded
+
+password = test &
+grant_type = password &
+client_secret = top-secret &
+client_id = wls &
+username = wls_all_uwb
+
+> {%
+    client.global.set("auth_token", response.body.access_token);
+    client.global.set("token_type", response.body.token_type);
+%}
+
+### get userinfo with auth_token
+GET {{ SSO_URL }}/auth/realms/wls_realm/protocol/openid-connect/userinfo
+Authorization: {{ token_type }} {{ auth_token }}
+
+### GET wahlvorstand fallback - working with dummy.client
+GET {{ WLS_WAHLVORSTAND_SERVICE_URL }}/businessActions/wahlvorstand/wahlbezirkID1
```

Comment:
Bitte auch ein Beispiel mit ForceUpdate=true ergänzen

Reply:
siehe commit [f6c13b1](https://github.com/it-at-m/Wahllokalsystem/pull/533/commits/f6c13b19d92d8668ae89eaacd0fb097c97177a8f)